_posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Pulp 3.10 is now Generally Available!
+author: Melanie Corr
+tags:
+  - release
+---
+
+We are happy to announce the release of Pulpcore 3.10!
+
+Let’s take a look at some of the main features of this release. For a list of bug fixes, see the [changelog](https://docs.pulpproject.org/en/3.10/changes.html)
+
+## Pre-validation import parameters
+
+A synchronous **import-check** command has been added to validate a number of possible causes of import/export failure. This command does not do tests that require database access or long-running tasks such as verifying checksums. All parameters are optional. [#7549](https://pulp.plan.io/issues/7549)
+
+For more information, check out the [Pulp Import and Export workflow](https://docs.pulpproject.org/pulpcore/en/3.10/workflows/import-export.html) documentation.
+
+## Update to the default Pulp deployment layout
+
+`MEDIA_ROOT` is now its own directory. This change has the advantage of limiting the file permissions and the SELinux file contexts in a shared Pulp 2 and Pulp 3 deployment. This also adds compatibility with the `django_extensions' unreferenced_files` command, which lists all files in `MEDIA_ROOT` that are not in the database.
+
+Other paths are kept on the same absolute paths.
+
+The settings and logging documentation has been updated to reflect these changes. [#7178](https://pulp.plan.io/issues/7178)
+
+## New generic list endpoint
+
+This release includes a new endpoint that you can use to list all `Content`, `ContentGuards`, and `Repositories`. [#7204](https://pulp.plan.io/issues/7204)
+
+## Update to the SigningService
+
+A new `public_key` field has been added to the **SigningService**. This field preserves the value of the public key. A new `pubkey_fingerprint` field was introduced as well, which identifies the
+public key. [#7700](https://pulp.plan.io/issues/7700)
+
+As a result of this, the following deprecation notice has been added for this release:
+
+`The path to the public key is no longer required to be present in the output of a signing script. It is recommended to pass the value of the public key during the object creation process instead.`
```

Comment:
```suggestion
`The path to the public key is no longer required to be present in the output of a signing script. It is required to pass the value of the public key during the object creation process instead.`
```

Reply:
For this blog @mdellweg I just took what message was in the code. The change would have to happen elsewhere?

Reply:
I'm seeing this one:
https://github.com/pulp/pulpcore/commit/2e0366013d2035c96156bf83debdfc7c03f647a0#diff-81f6a78175bb93934b6beff952646d3ca1ef3731f1ff14492d4ec77bfc3fdf82R652

Reply:
Will amend that now!